Code/DataStructs/BitOps.cpp

```diff
@@ -785,6 +785,7 @@ template void UpdateBitVectFromBinaryText(ExplicitBitVect&, const std::string&);
 #ifdef _MSC_VER
 #include <intrin.h>
 #define __builtin_popcount __popcnt
+#define __builtin_popcountl __popcnt64
```

Comment:
How portable is popcount across different compilers and CPUs? I think clang supports:
__builtin_popcount
__builtin_popcountll (note: 2 l's)

http://clang.llvm.org/doxygen/popcntintrin_8h_source.html

In addition, are we using the compiler flags:

g++ -mpopcnt
clang++ -march=corei7

but you may get into portability issues:

http://stackoverflow.com/questions/13774145/i-want-both-implementations-of-builtin-popcountll